### ISSUE TITLE
How to prevent the page footer overlap with table rows

### ISSUE THREAD

**Issue by lizet-sf** [clj-pdf/clj-pdf#232]
I have a footer in my pages like this
```
:footer {:table
          [:pdf-table
           {:border false}
           [20 15 60]
           ["this is a table footer" "second column" "third column"]
           ["this is a table footer" "second column" "third column"]
           ["this is a table footer" "second column" "third column"]
           ["this is a table footer" "second column" "third column"]]
```

And I have another table with several rows that don't fit in one page, so the other rows moves to next pages
My table looks like this
```
[:pdf-table {:width-percent 100
                     :border false
                     :header (header "title 1" "title 2" "title 3")}
         [15 50 25]
         ["row 1" "second column" "third column"]
         ["row 2" "second column" "third column"]
         ["row 3" "second column" "third column"]
         ["row..n" "second column" "third column"]]
```

The problem is that the last rows of my table in each page overlap with the text in the footer, 
Is there a way to prevent this to happen?

**Comment by yogthos** [comment]
It might be possible to use padding as a workaround here.

**Comment by lizet-sf** [comment]
Thanks, I already tried with padding, but documentation for pdf-table doesn't mention padding as metadata and the behavior is weird when I add it. 
Also I created a root table with one pdf-cell with a bottom padding and added my entire content to it. This partially solved my problem because I have extra content outside the table.

I am wondering if is there a way to set the height of the footer in order to the content can automatically jump to the next page.
